docs/dev-process/breaking-change-contract.md

```diff
@@ -72,6 +72,7 @@ Changes to surface area or behavior that is clearly internal or non-breaking in
 Examples:
 
 + Changes to internal APIs that break private reflection
```

Comment:
If you want, you can append something like "including all APIs in System.Management.Automation.Internal namespace."  Not required for this PR.

Reply:
Added note.